.github/workflows/deploy-emit.yml

```diff
@@ -43,7 +43,6 @@ jobs:
       - name: Update Data 📊
         working-directory: ./emit-ch4plume-v1
         run: |
-          mkdir ./data
```

Comment:
why remove?

Reply:
We are handling the data from webpack configuration. That line is not needed

Reply:
Also `data` folder is kept with .gitkeep